src/cascadia/TerminalApp/AppLogic.idl

```diff
@@ -109,7 +109,7 @@ namespace TerminalApp
         // See IDialogPresenter and TerminalPage's DialogPresenter for more
         // information.
         Windows.Foundation.IAsyncOperation<Windows.UI.Xaml.Controls.ContentDialogResult> ShowDialog(Windows.UI.Xaml.Controls.ContentDialog dialog);
-        void DismissDialog();

```

Comment:
This is weird, for some reason when I left this in I got the compiler error "ambiguous call to overloaded function" (definitely something to do with adding `DismissDialog` to the `IDialogPresenter` interface but I'm not sure why it only happens with `DismissDialog` and not `CanShowDialog`)

Reply:
You're right! Removed